app/uk/gov/hmrc/gform/gform/handlers/FormValidator.scala

```diff
@@ -91,24 +90,16 @@ class FormValidator(implicit ec: ExecutionContext) {
     validationResult.toFormValidationOutcome(enteredVariadicFormData)
   }
 
-  def fastForwardValidate(
+  def maybeGetInvalidSectionNumber(
```

Comment:
Why is this called `maybeGetInvalidSectionNumber`? Why e need invalid section number?

Reply:
I think this is `diff` shows the change as renaming. In fact I just created a new method `maybeGetInvalidSectionNumber` that checks if there is an invalid section number in range. `fastForwardValidate` is still there

Reply:
Still I have sort of problem with this name. 

Looking at the implementation it is not only about invalid section, is it? 
If that is right, can we rename this method, perhaps it can hint what the result of this method is intended to be used for?

Reply:
how about `def mustBeVisitedSectionNumber` ?